scripts/src/java/org/oppia/android/scripts/todo/TodoOpenCheck.kt

```diff
@@ -41,40 +48,28 @@ fun main(vararg args: String) {
   // Path to the JSON file containing the list of open issues.
   val openIssuesJsonFile = File(repoPath, args[2])
 
+  if (!openIssuesJsonFile.exists()) {
+    throw Exception("$repoPath${args[2]}: No such file exists")
+  }
```

Comment:
```suggestion
  check (openIssuesJsonFile.exists()) { "$repoPath${args[2]}: No such file exists" }
```

But more idiomatic. Ditto elsewhere for similar checks.

Reply:
Done.